config/routes.rb

```diff
@@ -2,11 +2,11 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
   get '/register', to: 'users#new'
-  post '/login', to: 'users#login'
-  post '/logout', to: 'users#logout'
+  post '/login', to: 'session#login'
+  post '/logout', to: 'session#logout'
```

Comment:
Since we're doing a sessions controller now, I would recommend renaming these to sessions#create and sessions#destroy since that's more RESTful

Reply:
Of course that also means changing it in the routes

Reply:
Fixed!